packages/manager/modules/pci/src/projects/project/storages/databases/components/calendar-edit/calendar-edit.controller.js

```diff
@@ -0,0 +1,57 @@
+import moment from 'moment';
+
+export default class PciCalendarEditController {
+  /* @ngInject */
+  constructor($timeout) {
+    this.$timeout = $timeout;
+  }
+
+  $onInit() {
+    // Compute GMT
+    const timezone = (new Date().getTimezoneOffset() / 60) * -1;
+    this.gmt = 'GMT';
+    if (timezone !== 0) {
+      this.gmt += timezone > 0 ? ' +' : ' ';
+      this.gmt += timezone;
```

Comment:
**suggestion:** These 2 statements can be combined.
```suggestion
      this.gmt += `${timezone > 0 ? ' +' : ' '}${timezone}`;
```

Reply:
https://github.com/ovh/manager/pull/7363/commits/e09e92ee34b574a021ba66ecca597aaf4724d05c